dockerfiles/image.sh

```diff
@@ -44,11 +44,17 @@ create_build_image() {
 
 create_platform_core_image() {
     local base=$1
-    echo "--- Create 'platform-core:$MODE' image ---"
+    echo "--- Create 'platform-core:release' image ---"
     docker build --rm -f "$SCRIPT_DIR/vcpkg/platform-core-$BUILD_OS.dockerfile" \
-        -t platform-core:$MODE \
+        -t platform-core:release \
         --build-arg BASE_IMAGE=$base \
             "$SCRIPT_DIR/vcpkg/." 
+
+    echo "--- Create 'platform-core:debug' image ---"
```

Comment:
question: now unconditionally building platform-core:release and platform-core:debug ?
Isn't that time consuming/wasteful?